server/routes/api/place.py

```diff
@@ -162,6 +160,40 @@ def statsvars(dcid):
     return response['places'][dcid].get('statsVars', [])
 
 
+@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+def get_stat_vars_union(dcids):
+    """Get all the statistical variable dcids for some places.
+
+    Args:
+        dcids: Place DCIDs separated by "^" as a single string.
+
+    Returns:
+        List of unique statistical variable dcids each as a string.
+    """
+    dcids = dcids.split("^")
+    # The two indexings are due to how protobuf fields are converted to json
```

Comment:
That's unfortunate... we should hide these complexities from the user. We should be able to remove that outer statVars. Please add a todo to update this.